onnxruntime/core/providers/nv_tensorrt_rtx/onnx_ctx_model_helper.h

```diff
@@ -20,21 +20,20 @@ static const std::string COMPUTE_CAPABILITY = "hardware_architecture";
 static const std::string ONNX_MODEL_FILENAME = "onnx_model_filename";
 static const std::string EPCONTEXT_OP_DOMAIN = "com.microsoft";
 static const std::string EPCONTEXT_WARNING =
-    "It's suggested to set the ORT graph optimization level to 0 and  \
-                                              make \"embed_mode\" to 0 (\"ep_cache_context\" is the cache path)\
-                                              for the best model loading time";
+    "It's suggested to set the ORT graph optimization level to 0 for the best performance";
```

Comment:
Not just for best performance. We want to circumvent that initializers get constant folded in the optimizer and therefore change their name.